### PR TITLE
Apple Home page clone

### PR DESCRIPTION
**Pull request by Nkuria**
This is a clone for the previous apple home page created using HTML and CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+## Authors
+
+👤 **Nick Kuria**
+
+- Github: [@githubhandle](https://github.com/Nkuria)
+
+👤 **Mina Ibrahim**
+
+- Github: [@githubhandle](https://github.com/coldatlas)
+
```
Comment (SpaYco) [inline README.md]: - [x] [Required] your README can be more descriptive ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
 also change the '@githubhandle' to your actual handle for example mine is @Spayco, you can also add the rest of your social media profiles, but that's optional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <button type="button" id="applebutton">
+          <i class="fab fa-apple"></i>
+        </button>
+        <button type="button" id="storebutton">Store</button>
+        <button type="button" id="macbutton">Mac</button>
+        <button type="button" id="ipodbutton">iPod</button>
+        <button type="button" id="iphonebutton">iPhone</button>
+        <button type="button" id="ipadbutton">iPad</button>
+        <button type="button" id="itunesbutton">iTunes</button>
+        <button type="button" id="supportbutton">Support</button>
```
Comment (SpaYco) [inline index.html]: - [x] [Required] The first eight elements should be links (to other sections or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <input type="text" id="searchbutton" /><i class="fas fa-search"></i>
```
Comment (SpaYco) [inline index.html]: - [x] [Required] your search icon is flying outside of the input, please fix it ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

image

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)), you have some images that are not used

- I've included more changes down below

## Optional 

- [x]  use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        </div>
+
+      </nav>
+      <a href="#"><img src="./images/your_verse_title_small.png" alt="#" /> </a>
+      <a href="#"
+        ><img src="./images/your_verse_subhead_small.png" alt="#"
+      /></a>
+      <br />
+      <a href="#"><img src="./images/cta_your_verse_27pt.png" alt="#" /></a>
+    </header>
```
Comment (santiagorodriguezbermudez) [inline index.html:37]: - [x] These should not be images but rather h and p tags elements.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  left: 5px;
+  bottom: 3px;
+  color: white;
+}
+
+header {
+  background: url("./images/apple2.jpg");
+  background-repeat: no-repeat;
+  min-height: 500px;
+  background-size: 100% auto;
```
Comment (santiagorodriguezbermudez) [inline style.css]: - [x] Change background-size to 'cover'. This way the image will always occupy all of its backgrounds.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <header>
+      <nav class="nav-bar">
+        <a href="#"><div id="applebutton"><i class="fab fa-apple"></i></div></a>
+        <a href="#"><div id="storebutton">Store</div></a>
+        <a href="#"><div id="macbutton">Mac</div></a>
+        <a href="#"><div id="ipodbutton">iPod</div></a>
+        <a href="#"><div id="iphonebutton">iPhone</div></a>
+        <a href="#"><div id="ipadbutton">iPad</div></a>
+        <a href="#"><div id="itunesbutton">iTunes</div></a>
+        <a href="#"><div id="supportbutton">Support</div></a>
```
Comment (santiagorodriguezbermudez) [inline index.html]: - [x] These ids are not being used. Kindly delete them. Often, prefer use classes over ids.

**Review by santiagorodriguezbermudez — CHANGES_REQUESTED** [review]
# Changes Required ♻️

Good job implementing the last changes from your last CR. However, there are still a couple of changes you must implement. 

## Required Changes 🤓
- [x] Fixed the padding of your navbar. Moreover, fixed the width of your navbar as well. It should be wider according to the original page. 
Screen Shot 2020-08-20 at 6 27 18 AM

- [x] The right upper search input element should be embedded inside the element and again a padding and background shadow should be added. 
Screen Shot 2020-08-20 at 6 28 48 AM

- [x] The four images on your page have a white space in between them that is currently not in the original page. Kindly, remove this white space. 
Screen Shot 2020-08-20 at 6 29 55 AM

- [x] The footer looks messy and missing some bars in between them. As well as the links in their color blue. Again, review the original page. 
Screen Shot 2020-08-20 at 6 32 34 AM

- [x] Your Pull Request must have more information. Every time you provide for new changes, write a descriptive comment of the changes you've implemented since the last CR. This will help you communicate better your changes. 

- [x] Changes over your code (see below)

**Please implement the mentioned changes and submit another code review 🧐**

Happy coding 🙂,
Cheers,
Santiago Rodriguez B.
contact me on slack (@santiagorodriguezb) if you need any help.

**Comment by m-anwar97** [comment]
Fixed the padding of the navbar.
Added the search element inside the nav bar.
Removed the white spaces between the four images.
Added bars between footer elements, added blue links just like the original page.
Added this pull request comment.
Made the required changes to the code asked by the reviewer

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

Hi team,  good job on making the previously required changes!
You guys are almost there, but in order to accomplish as the actual page, you are required to do these changes listed below:

## [REQUIRED CHANGES]
- [x] I still can see a white space at the end of those 4 images. That is currently not on the original page. Kindly, remove this white space. 
image

- [x] Please improve the footer 🦶, as requested the previous TSE, to make it more similar to the original one.
* Yours
image
* Original
image




## [OPTIONAL CHANGES]
 - [ ] Improve your navbar, make it more similar to the original one
* Yours
image
* Original one
image
- [ ] Put your `css` files in a css folder. (To make your repo more professional)


## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by m-anwar97** [comment]
Fixed the white space issue and improved footer

**Review by kitpao — APPROVED** [review]
# 👋 Hello @coldatlas and @Nkuria 

🌟 Great job implementing your last mandatory changes, now it is working as expected 👏 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**